freshmaker/handlers/koji/rebuild_flatpak_application_on_module_ready.py

```diff
@@ -167,3 +179,152 @@ def _filter_images_with_higher_rpm_nvr(self, event, rebuild_images_list):
         )
         images = lb.get_images_by_nvrs(rebuild_images_list, rpm_nvrs=errata_rpm_nvrs)
         return images
+
+    def _get_module_name_stream_set(self, event):
+        """
+        Get set of modules with format NAME:STREAM in an advisory.
+
+        :param event FlatpakModuleAdvisoryReadyEvent: The event this handler
+            is currently handling.
+        :return: a set with item NAME:STREAM
+        :rtype: set
+        """
+        errata = Errata()
+        module_nvrs = errata.get_cve_affected_build_nvrs(event.advisory.errata_id, True)
+
+        module_name_stream_set = set()
+        with koji_service(
+            conf.koji_profile, log, login=False, dry_run=self.dry_run
+        ) as session:
+            for module_nvr in module_nvrs:
+                mmd = session.get_modulemd(module_nvr)
```

Comment:
Actually I was confused with the [doc](https://fedora-modularity.github.io/libmodulemd/latest/modulemd-2.0-Modulemd.html#modulemd-read-packager-string) too. But when I was testing it, the returned value is a _ResultTuple like "(<GType ModulemdModuleStreamV2 (94271507489296)>, object=<Modulemd.ModuleStreamV2 object at 0x7fe3eea26700 (ModulemdModuleStreamV2 at 0x55bd4bb78940)>)"

Reply:
I forgot this, could you try it in our freshmaker image? I guess it could be caused by some different version of packages, but I didn't dig into this, error from what I tried with `quay.io/factory2/freshmaker:prod`:
```
>  _, mmd = Modulemd.read_packager_string(modulemd_str, None, None)
                                                                                                                         
TypeError: cannot unpack non-iterable ModuleStreamV2 object
```

Reply:
Ack. Fixed.
Add root cause for others, seems rpm package install is different with pip install, rpm will provide overrides which will help to unpack and strip GType, but pip install will not, so the return value is different.